apps/www/_blog/2024-08-16-postgres-foreign-data-wrappers-with-wasm.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: 'Postgres Foreign Data Wrappers with Wasm'
+description: 'Build Wasm foreign data wrapper with Wrappers and use it on Supabase'
+author: bo_lu
+image: lw12/day-5/wasm-fdw-og.png
+thumb: lw12/day-5/wasm-fdw-thumb.png
+launchweek: '12'
+categories:
+  - launch-week
+  - developers
+  - product
+tags:
+  - launch-week
+  - wasm
+  - wrappers
+date: '2024-08-16'
+toc_depth: 3
+---
+
+## Introduction
+
+Foreign Data Wrappers (FDWs) allow Postgres to interact with externally hosted data. To operate a FDW, the user creates a foreign table. When queried, the foreign table reaches out to the 3rd party service, collects the requested data, and returns it to the query in the shape defined by the foreign table. This allows seamless querying and data manipulation across different tools as if they were local tables from within Postgres.
+
+[Wrappers](https://github.com/supabase/wrappers) is a Rust framework for creating Postgres Foreign Data Wrappers. Today we’re releasing support for [Wasm (WebAssembly)](https://webassembly.org/) wrappers.
```

Comment:
Can we move this part to the top? It would be good not to bury the lede

Reply:
I just removed the `Introduction` heading.